doc/sphinx/language/core/modules.rst

```diff
@@ -1010,3 +1010,66 @@ subdirectories of path). See the command :cmd:`Declare ML Module` in
 
 See :ref:`command-line-options` for a more general view over the Coq command
 line options.
+
+.. _controlling-locality-of-commands:
+
+Controlling the scope of commands with locality attributes
+----------------------------------------------------------
+
+Many commands produce an effect that will persist in a given scope,
+which can typically be the one of the section or of the module in
+which the command was called. Locality :term:`attributes <attribute>`
+can be used to change the scope of this effect. Note that some
+commands, like :cmd:`Arguments` do not have the same default locality
+when they are used inside or outside sections.
```

Comment:
```suggestion
Many commands have effects that apply only within a specific scope,
typically the section or the module in
which the command was called. Locality :term:`attributes <attribute>`
can alter the scope of the effect. Note that some
commands, like :cmd:`Arguments` do not have the same default locality
when they are used inside or outside sections.
```

"typically"?  what other scopes are there other than sections and modules?
"Note that .." should be under `attr:: local`.  In this paragraph you could say
that a few commands don't follow the general pattern--that alerts the reader
that quirks lie ahead.

Reply:
> "typically"? what other scopes are there other than sections and modules?

Some commands have effects which persist module boundaries by default.

I've added an alert. Let me know what you think.